src/airstack/execute_query.py

```diff
@@ -202,7 +202,7 @@ async def get_next_page(self, query, variables, page_info):
                     self.deleted_queries.append(None)
                 if has_cursor(document_ast, _page_info_key):
                     replace_cursor_value(document_ast, _page_info_key,
-                    _page_info_value['nextCursor'])
+                    _page_info_value['nextCursor'], self.variables)
```

Comment:
Should we not throw the error if `_page_info_value['nextCursor']` is an empty string?